src/main/webapp/app/shared/functions/query-params.utils.ts

```diff
@@ -0,0 +1,10 @@
+
+const buildQueryParamString: (...params: string[]) => string = (...params: string[]): string => {
+    return params.reduce((total, next) => (total + (next ? `${next}&` : '')), '').slice(0, -1);
```

Comment:
Why are you not using standard Angular functionality with the `httpClient`'s build-in arguments?

Reply:
First, I created that function for the local-task.service.ts, so I could replicate as much as possible an api call (with params, building, creating, etc), and that is why I remained set on that :)). No reason not to use the httpClient arguments.

Reply:
Deleted this unused utility, and integrated httpClient -> options -> params: HttpParams. Changes present in the last commit